src/BioentityInformation.js

```diff
@@ -67,7 +69,7 @@ class BioentityInformation extends React.Component {
   render() {
     const bioentityProperties = this.state.bioentityProperties.map((property) =>
       <tr key={property.type}>
-        <td>
+        <td className={`gxaBioentityInformationCardPropertyType`}>
```

Comment:
Please, add CSS modules to the package and we can get rid of the `gxa` prefix for class names. Remnants of old times when we didn’t know any better. ;)